### ISSUE TITLE
IE11 Syntax Error (Fat Arrow) with Webpacker default install

### ISSUE THREAD

**Issue by MrSkinny** [rails/webpacker#1837]
IE11 is reporting a syntax error here:
```
Object.keys(Methods).forEach((methodName) => {
  __WEBPACK_IMPORTED_MODULE_0_dom7_dist_dom7_modular__["a" /* $ */].fn[methodName] = Methods[methodName];
});
```

It doesn't support fat arrows. 

This is occurring using all defaults from current stable version install of `gem "webpacker", "~> 3.5"`

I have tried changing `.babelrc` by setting browsers to `"> 0.25%, not dead"` and adding `"transform-es2015-arrow-functions"` plugin.  (Full `.babelrc` below)  

I am importing `@babel/polyfill` as the very first line in my pack's `index.js`.

Any other suggests MUCH appreciated!!

```
{
  "presets": [
    ["env", {
      "modules": false,
      "targets": {
        "browsers": "> 0.25%, not dead",
        "uglify": true
      },
      "useBuiltIns": true
    }]
  ],

  "plugins": [
    "syntax-dynamic-import",
    "transform-object-rest-spread",
    ["transform-class-properties", { "spec": true }],
    "transform-es2015-arrow-functions"
  ]
}
```

**Comment by gauravtiwari** [comment]
Can you create a test repo, please? Looks alright to me, maybe something to do with the way things are set up.

**Comment by MrSkinny** [comment]
Well, I found it. In fact I found two. Am using the [Swiper](https://www.npmjs.com/package/swiper) library and that is causing the exact syntax error. I can post an issue there although it seems like this is something for Webpack/Babel since any given code should not be able to prevent transpiling to ES5 right?

I also have the use of the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) api in my project which is causing an "Object does not support this action" error. This is showing not compat in IE11, and I guess polyfill doesn't help...

**Comment by gauravtiwari** [comment]
@MrSkinny Could you try latest RC, please? It has support for transpiling node modules.

**Comment by MrSkinny** [comment]
@gauravtiwari I've resolved by pointing to the `dist` files in Swiper, but happy to try importing node modules on RC.  

I've done the following steps, and am still getting the same issue.  I think my webpack isn't upgrading; how do I force this?

1) Updated gemfile to `gem 'webpacker', '4.0.0.rc.2'
2) Ran `bundle update webpacker`
3) Ran `yarn install`
4) Made arbitrary change to pack entry js to force a new js compile

The new compile is still showing fat arrows.  When I run `bin/webpack --version` I get 3.12.0.  Assume this should be 4.x?  I tried regenerating the binstubs with `rails webpacker:binstubs` and it asked me to overwrite `bin/webpack` and `bin/webpack-dev-server` which I confirmed.  But when I still get 3.x when checking version...
